starter-code/pizza.js

```diff
@@ -1 +1,24 @@
 // Write your Pizza Builder JavaScript in this file.
```

Comment:
I see what you did here.  Good idea to make a generic function and apply it to all the different toppings.  Only 1 problem, when someone click the gluten-free-crust button, it takes away the whole bread underneath the pizza.  We don't need to toggle the crust, we just need to add/remove the crust-gluten-free class and it will change back to a regular crust.